tests/testthat/test-href.R

```diff
@@ -101,3 +101,16 @@ test_that("or local sites, if registered", {
   scoped_package_context("pkgdown", local_packages = c("digest" = "digest"))
   expect_equal(href_expr_(vignette("sha1", "digest")), "digest/articles/sha1.html")
 })
+
+test_that("github_source linking", {
+  base <- "https://github.com/r-lib/pkgdown"
+  expect_silent(
```

Comment:
Can you drop the `expect_silent()` please?

Reply:
Sure; this was part of my failing test because the initial symptom of the issue was a `warning`.